### PR TITLE
add basic spectrum example for RFmxSpecAn

### PR DESCRIPTION
**Pull request by gregstoll**
### What does this Pull Request accomplish?

Adds a basic spectrum example for RFmxSpecAn, modeled after a C example.

### Why should this Pull Request be merged?

We haven't decided exactly what examples we are going to write, but this is a good one to start with (and a good proof point that the API works)

### What testing has been done?

Ran it on my machine, and I get the results:

> min frequency: 999.500 MHz: -51.0 dBm
> midpoint frequency: 1.000 GHz: -8.5 dBm
> max frequency: 1.001 GHz: -51.0 dBm

which lines up with the graphs seen in the LabVIEW and C examples.

### CODE REVIEW

File: examples/nirfmxspecan/spectrum-basic.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            )
+        )
+    )
+
+    raise_if_error(
+        client.SpectrumCfgRBWFilter(
+            nirfmxspecan_types.SpectrumCfgRBWFilterRequest(
+                instrument_handle=instr, selector_string="",
+                rbw_auto=nirfmxspecan_types.SPECTRUM_RBW_AUTO_BANDWIDTH_TRUE,
+                rb_w=100e3,
```
Comment (astarche) [inline examples/nirfmxspecan/spectrum-basic.py:115]: (Not necessarily for this PR): Looks like we need to fix this param name. Not sure what happens when this goes through the various transformations. But I think the fix is probably: "RBW" -> "rbw" in the metadata.

File: examples/nirfmxspecan/spectrum-basic.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        f /= 1000.0
+    return f"{f:.3f} G{suffix}"
+
+
+# Raise an exception if an error was returned
+def raise_if_error(response):
+    if response.status != 0:
+        error_response = client.GetError(
+            nirfmxspecan_types.GetErrorRequest(
+                instrument_handle=instr,
```
Comment (astarche) [inline examples/nirfmxspecan/spectrum-basic.py:75]: (Not necessarily for this PR): This `instrument_handle` name seems too long. The obvious thing is to drop handle and go with just `instrument`, which is what we did for DAQ `taskHandle`s. That seems pretty good. `instr` is appealing too, but it's better to err on the side of complete words these days. Any other ideas?

File: examples/nirfmxspecan/spectrum-basic.py (hunk 1 above)
Comment (gregstoll) [reply examples/nirfmxspecan/spectrum-basic.py:115]: Yeah, the metadata has the parameter as "rbW". Filed #436 for this.

File: examples/nirfmxspecan/spectrum-basic.py (hunk 2 above)
Comment (gregstoll) [reply examples/nirfmxspecan/spectrum-basic.py:75]: Yeah, I like `instrument` - filed #437.

**Pull request merged**